webm/webm.go

```diff
@@ -22,6 +22,7 @@ import (
 
 // EBMLHeader represents EBML header struct
 type EBMLHeader struct {
+	ebml.Metadata
```

Comment:
I think it would be better to be optional and on-demand.

Reply:
Do you mean all `Metadata`s should be optional by user? or only the EBMLHeader's Metadata?

Reply:
Yes, I think the pre-defined structs in webm package is better not to have Metadata field since `webm.FrameWriter` don't use `Metadata`. Also, I think the struct is better to be able to be roundtripped by default.

Reply:
Got it, so I reconsider its design and post it to #45